tests/test_smoke.py

```diff
@@ -797,6 +797,28 @@ def test_gcp_mig():
     run_one_test(test)
 
 
+@pytest.mark.gcp
+def test_gcp_force_enable_external_ips():
+    name = _get_cluster_name()
+    test_commands = [
+        f'sky launch -y -c {name} tests/test_yamls/minimal.yaml',
```

Comment:
Let's specify the `--cloud gcp` to avoid the cluster get provisioned on other clouds. (also, included a `--cpus 2` to save some cost for testing)
```suggestion
        f'sky launch -y -c {name} --cloud gcp --cpus 2 tests/test_yamls/minimal.yaml',
```